lib/compiler/utils.js

```diff
@@ -50,3 +50,41 @@ function regexpClassEscape(s) {
     .replace(/[\u1000-\uFFFF]/g,      ch => "\\u"  + hex(ch));
 }
 exports.regexpClassEscape = regexpClassEscape;
+
+/**
+ * Base64 encode a Uint8Array.  Needed for browser compatibility where
+ * the Buffer class is not available.
+ *
+ * @param {Uint8Array} u8 Bytes to encode
+ * @returns {string} Base64 encoded string
+ */
+function base64(u8) {
+  // Note: btoa has the worst API, and even mentioning Buffer here will
+  // cause rollup to suck it in.
+
+  // See RFC4648, sec. 4.
```

Comment:
Direct link for pedants :)
```suggestion
  // See RFC4648, sec. 4.
  // https://datatracker.ietf.org/doc/html/rfc4648#section-4
```